ign-rviz/package.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<package format="3">
```

Comment:
```suggestion
<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
<package format="3">
```

Reply:
Colon test `xmllint` fails on adding this. As the project name has a hyphen character.
This is the same for all the packages.

Reply:
name is ROS 2 package can not contain `-`. You should rename them `ign_rviz` and `ign_rviz_plugins`

Reply:
I am a little confused about the programming conventions. As far as I know, ROS/ROS2 generally follows the `snake_case` for project_name, file_names, functions, etc. and use `PascalCase` for ClassNames.
Ignition, on the other hand, follows `camelCase / PascalCase` along with hyphen (`-`) character for the project-name.

Can we discuss the convention to be followed at the beginning of the project? It will be difficult to refactor the code later on.